code-for-canada/project-thundercat
# [Organizational Structure] Populating organizational structure tree view popup with db data

# Description

This PR is replacing localized variables in organizational structure image description popup with DB data.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**English Version:**

![image](https://user-images.githubusercontent.com/23021242/60668830-df1ca900-9e3a-11e9-8899-d5a699a4279f.png)

**French Version:**

![image](https://user-images.githubusercontent.com/23021242/60668843-e9d73e00-9e3a-11e9-89dd-367ab8737132.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Click on Background tab.
3. Click on _Organizational structure_ on side nav.
4.  Open the image description.
5.  Make sure that the popup content looks exactly the same as before in both languages.

# Checklist

Applicable for all code changes.
